entrypoint

```diff
@@ -33,7 +38,7 @@ fi
 if [ -n "$INPUT_ANALYZER" ]; then
     if [ "$INPUT_ANALYZER" = "clang-tidy" ]; then
         install_dependencies
-        cmake -DCMAKE_EXPORT_COMPILE_COMMANDS=ON .
+        "${cmake[@]}" . -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
```

Comment:
I'm not so sure about this one... Which use case are you targeting?

Reply:
I was actually just copying that over for the sake of conformity, I don't actually have a use-case for that.  I deleted it in the latest push.